images/runtime/node/8/base.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM oryx-node-run-base
+FROM oryx-node-run-base-stretch
```

Comment:
>stretch [](start = 24, length = 7)

Why are these versions locked to stretch only? Is it because they are going to be deprecated anyway?

Reply:
yepp